test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -6,9 +6,34 @@
             [puppetlabs.puppetserver.testutils :as testutils]
             [puppetlabs.trapperkeeper.testutils.webserver :as jetty9]
             [puppetlabs.services.master.master-core :as master-core]
-            [puppetlabs.services.protocols.jruby-puppet :as jruby]
+            [puppetlabs.services.protocols.jruby-puppet :as jruby-protocol]
             [cheshire.core :as cheshire]
-            [me.raynes.fs :as fs]))
+            [me.raynes.fs :as fs]
+            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
+            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
+            [clojure.tools.logging :as log]
+            [puppetlabs.trapperkeeper.core :as tk]
+            [puppetlabs.services.protocols.puppet-server-config :as
+             ps-config-protocol]
+            [puppetlabs.services.ca.ca-testutils :as ca-testutils]
+            [puppetlabs.services.master.master-service :as master-service]
+            [puppetlabs.services.request-handler.request-handler-service :as
+             handler]
+            [puppetlabs.services.jruby.jruby-puppet-service :as jruby-service]
+            [puppetlabs.services.puppet-profiler.puppet-profiler-service :as
+             profiler]
+            [puppetlabs.trapperkeeper.services.webserver.jetty9-service :as
+             webserver]
+            [puppetlabs.trapperkeeper.services.webrouting.webrouting-service
+             :as webrouting]
+            [puppetlabs.services.puppet-admin.puppet-admin-service :as admin]
+            [puppetlabs.services.ca.certificate-authority-service :as ca]
+            [puppetlabs.trapperkeeper.services.authorization.authorization-service
+             :as authorization]
+            [puppetlabs.services.versioned-code-service.versioned-code-service
+             :as vcs])
```

Comment:
Yeah, I hate doing this too.  We did something like this with one of the tk-auth integration tests a while back, too.  If we're seeing more of a need to do testing where we're using most of the normal service stack but with a mocked version of the jruby-puppet-service, maybe creating a `with-puppetserver-and-mocked-jruby-service` macro would be a good thing to create at some point?  Would be nice to avoid having to proliferate knowledge of what the full service stack is out to lots of different integration test namespaces.